cmd/merge_test.go

```diff
@@ -49,9 +49,11 @@ func Test_MergeCmd_Run_good(t *testing.T) {
 		cmd      MergeCmd
 		rowCount int64
 	}{
-		"good":    {MergeCmd{rOpt, wOpt, 10, []string{"good.parquet", "good.parquet"}, "", false}, 6},
-		"empty":   {MergeCmd{rOpt, wOpt, 10, []string{"empty.parquet", "empty.parquet"}, "", false}, 0},
-		"top-tag": {MergeCmd{rOpt, wOpt, 10, []string{"top-level-tag1.parquet", "top-level-tag2.parquet"}, "", false}, 6},
+		"good":                    {MergeCmd{rOpt, wOpt, 10, []string{"good.parquet", "good.parquet"}, "", false}, 6},
+		"random-example-repeated": {MergeCmd{rOpt, wOpt, 10, []string{"random-example1.parquet", "random-example1.parquet"}, "", false}, 54},
```

Comment:
Would you please use `all-types.parquet` instead? or if you really want to use `random-example1.parquet`, please help to add a small program to generate this test data, there are quite some examples under https://github.com/hangxie/parquet-tools/tree/main/testdata/gen.

Reply:
Yep all tpyes works that triggered the test case